mobile/src/main/java/ch/epfl/sdp/mobile/ui/prepare_game/ColorChoiceTabBar.kt

```diff
@@ -0,0 +1,94 @@
+package ch.epfl.sdp.mobile.ui.prepare_game
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.selection.selectable
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.drawBehind
+import androidx.compose.ui.graphics.*
+import androidx.compose.ui.graphics.drawscope.Stroke
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import ch.epfl.sdp.mobile.state.LocalLocalizedStrings
+import ch.epfl.sdp.mobile.ui.*
+
+enum class TabColors(val backgroundColor: Color, val textColor: Color, val borderColor: Color) {
+  SelectedColors(PawniesColors.Green800, PawniesColors.Orange200, PawniesColors.Orange200),
+  UnselectedColors(PawniesColors.Green100, PawniesColors.Green800, PawniesColors.Green100)
+}
+
+@Composable
+fun ColorChoiceBar(
+    colorChoice: ColorChoice,
+    onColorChange: (ColorChoice) -> Unit,
```

Comment:
(minor) I'd perhaps name this `onSelectColor` to clearly reflect the user action.

Reply:
Agree